infrastructure/gke-billing-export/README.md

```diff
@@ -0,0 +1,124 @@
+# gke-billing-export
+
+Google Kubernetes Engine fine grained billing export. This consists of a single binary that can be run in a Google Cloud project, which with appropriate permissions can scrape the metrics server for many Kubernetes Engine clusters. The metrics are stored in a dedicated BigQuery dataset.
+
```

Comment:
Things I'd suggest adding to the readme to help users understand what's provided:
- what's included in the config file?
- mention that the binary is `billing` since this is different from go default of using directory name
- how the tool is built with docker and deployed with deployment manager

Here's a starting point:

The executable takes no arguments, it reads project id and polling interval from  [config.json](config.json) and runs continuously as a daemon.
This repository provides a docker file for creating an image with the executable baked in, as well as a deployment manager configuration to run the image on your GKE cluster.
Once deployed, the billing export daemon will periodically poll the metrics API and save results to BigQuery.
```

Reply:
Done, thanks!